modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -200,7 +205,12 @@ class NDB_Form_user_accounts extends NDB_Form
 
         ///get the value for additional_user_info flag
         $config =& NDB_Config::singleton();
-        $additional_user_info = $config->getSetting('additional_user_info');
+        try {
```

Comment:
I think that the try/catch should be there. The value could be missing from
the config/DB for various reasons. I should maybe display en error page
indicating that the setting is missing instead of silently doing as if it
was set to false though. What do you think ?


Nick


On Wed, May 6, 2015 at 2:15 PM, Dave MacFarlane <notifications@github.com>
wrote:

> In modules/user_accounts/php/NDB_Form_user_accounts.class.inc
> <https://github.com/aces/Loris/pull/1090#discussion_r29785885>:
>
> > @@ -200,7 +205,12 @@ class NDB_Form_user_accounts extends NDB_Form
> >
> >          ///get the value for additional_user_info flag
> >          $config =& NDB_Config::singleton();
> > -        $additional_user_info = $config->getSetting('additional_user_info');
> > +        try {
>
> This shouldn't have a try/catch block.. if the setting is missing, it
> needs to be added to the schema.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aces/Loris/pull/1090/files#r29785885>.
>